tests/markdown_link/title.md

```diff
@@ -1 +1,3 @@
 [hello](#world "title")
+[hello](#world 'title')
+[hello](#world (title))
```

Comment:
Can you add non-trivial tests with escapes and stuff as well?

Reply:
Would just the `\'`, `\"`, and `\)` escapes be OK, or should I test other escapes?

Reply:
You should look at the CommonMark spec, see how the escapes work, add tests for those, and also add in things that _look_ like escapes but aren't and make sure that they aren't touched.

Reply:
✅ See https://github.com/prettier/prettier/pull/3481/commits/118fee862e121f5496a1ec95731e811926454b17.